src/main/kotlin/io/github/paulgriffith/kindling/zip/ZipView.kt

```diff
@@ -148,8 +150,18 @@ class ZipView(path: Path) : ToolPanel("ins 6, flowy") {
         }
 
         add(bundleInfo, "split 2, height 32!")
-        add(FlatScrollPane(fileTree), "pushy, growy, width 250!")
-        add(tabStrip, "newline, push, grow, spany")
+        add(
+            JSplitPane(
+                JSplitPane.HORIZONTAL_SPLIT,
+                FlatScrollPane((fileTree)),
+                tabStrip,
+            ).apply {
+                dividerSize += 2
```

Comment:
Just padding to make it easier to grab?

Reply:
Yeah lol. OneClickExpand arrows don't have any space around them and it looks tacky. This just puts more space there.